RELEASE_SCHEDULE.md

```diff
@@ -9,7 +14,72 @@ You can determine what is in a given build using the [releases page](https://git
 
 Announcements regarding releases will be made to the #headerbidding-dev channel in subredditadops.slack.com.
 
-# FAQs
+## Release Process
+
+1. Make Sure all browserstack tests are passing. Check the last travis build for master branch. If failing please fix the failing tests or skip the test and create issue if it takes too long to fix the failed unit test.
+
+   #### How to run tests in browserstack
```

Comment:
Does the coordinator need to run the tests manually or can they just look at the latest travis build? Row 19 says to check Travis -- do we just re-run the tests anyhow?

Reply:
Added more details.